packages/components/datagrid/README.md

```diff
@@ -32,6 +32,7 @@ angular.module('myModule', ['oui.datagrid'])
 | `rows`                            | array     | <?        | yes                 | n/a              | n/a          | local rows to load in the datagrid
 | `empty-placeholder`               | string    | @?        | yes                 | n/a              | n/a          | custom placeholder text when there is no data
 | `selectable-rows`                 | boolean   | <?        | no                  | `true`, `false`  | `false`      | allow rows to be selected
+| `expandable-rows`                 | boolean   | <?        | no                  | `true`, `false`  | `false`      | allow rows to be expanded when clicked
```

Comment:
Doesn't seem useful though, if you have a `oui-datagrid-row-detail`, then you know that the rows have a detail :)